### PR TITLE
Improve mining dispatcher

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

**Comment by polarker** [comment]
@h0ngcha0 @Lbqds @tdroxler Please help me review.

The idea is to use a standalone thread dispatcher for mining related actors, so they could have lower latency.

File: flow/src/main/resources/system_prod.conf.tmpl [hunk 1]
```diff
@@ -132,6 +132,7 @@ akka {
       executor = "fork-join-executor"
       fork-join-executor {
         parallelism-min = 1
+        parallelism-max = 4
```
Comment (polarker) [inline flow/src/main/resources/system_prod.conf.tmpl:135]: `ViewHandler` and `MiningApiController` will use one thread each, so a `max` of 4 threads seems fine

**Review by Lbqds — APPROVED** [review]
LGTM

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

**Pull request merged**